solana-mobile/mobile-wallet-adapter
# [WIP] New Architecture: mobile-wallet-adapter-protocol

Backwards compatible Turbo Module for `mobile-wallet-adapter-protocol`.

## Test plan
For each test case, we test `newArchEnabled` flag both `true/false` and verify MWA works.

### Test cases
1. Verify working integration for local Example App ✅ 
2. Publish `beta` version on NPM and verify it works in new RN project (test multiple versions <= 0.76.2) ❌ 
3. Publish `beta` version on NPM and verify it works in new Expo project (test multiple versions <=Expo SDK 52) ❌ 

Don't merge until testing is complete
